tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -380,25 +380,30 @@ tensorflow::Status CreateBroadcastableScalarConstant(
   return Status::OK();
 }
 
-// Convert an axis from TF format to TRT format while validating.
-tensorflow::Status ConvertAxis(int tf_axis, int trt_nb_dims, string node_name,
-                               int* trt_axis, bool allow_batch_dim = false) {
+// Convert an axis from TF format to TRT format while validating. TF format
+// includes the batch dimension, while TRT does not. TF can also use negative
+// indices.
+// TODO(tmorris): Use this method in more ops.
+tensorflow::Status ConvertAxis(int tf_axis, int trt_nb_dims, bool check_bounds,
```

Comment:
when do we expect us to set check_bounds to false?

Reply:
For Ops like ExpandDims which have different bounds than other ops like I mentioned earlier.

Reply:
I interpreted by comment that we wanted to handle that in place as it was just specific to ExpandDims. Regardless, adding that when we are actually using it would be better.

Reply:
Okay sounds good. I will remove the arg for now.